src/components/data/toolbar/DataDotMenu.js

```diff
@@ -160,14 +164,6 @@ function DataDotMenu(props) {
                                       <ListItemText primary={t("folder")} />
                                   </MenuItem>
                               ),
-                              renameEnabled && (
```

Comment:
I might be missing something but is there a reason why the rename menu item was removed?